framework/test/integration/process_transactions.js

```diff
@@ -55,6 +54,12 @@ describe('blocks/verifyTransactions', async () => {
 		senderId: accountFixtures.genesis.address,
 	}));
 
+	// If we include second signature transaction, then the rest of the transactions in the set will be required to have second signature.
+	// Therefore removing it from the appliable transactions
+	const appliableTransactions = verifiableTransactions.filter(
```

Comment:
typo I think change to `applicableTransactions `

Reply:
It's a grammatically correct word. I did google search prior to using it. :) https://www.thefreedictionary.com/appliable